tests/unit-tests/input/test_default_input_manager.cpp

```diff
@@ -16,6 +16,7 @@
  * Authored by: Andreas Pokorny <andreas.pokorny@canonical.com>
  */
 
+#include <memory>
 #include "src/server/input/default_input_manager.h"
```

Comment:
Please put `#include <memory>` after `#include <list>`, not before the project headers.